cli/popper/commands/cmd_env-vars.py

```diff
@@ -0,0 +1,80 @@
+import click
+import popper.utils as pu
+
+from popper.cli import pass_context
+from popper.exceptions import UsageError
+
+
+@click.command('matrix', short_help='Define execution matrix for a '
+               'pipeline using environment variables.')
```

Comment:
should we rename to `env-vars`? Another alternative I was thinking was `jobs`, where a "job" is defined as an execution of the pipeline on a set of env vars. Then, we'll have as many jobs as number of variable sets. Another one would be `executions`, since there's one execution of the pipeline for each variable set.

Reply:
env-vars sounds the most appropriate.

Reply:
Sounds good